ext/User.go

```diff
@@ -19,6 +19,6 @@ type UserProfilePhotos struct {
 }
 
 // GetProfilePhotos Retrieves a user's profile pictures
-func (user User) GetProfilePhotos(offset int, limit int) (*UserProfilePhotos, error) {
```

Comment:
https://core.telegram.org/bots/api#getuserprofilephotos
If youre removing the offsets for the base image, might be worth adding a `GetProfilePhotosOffset(offset int, limit int) which allows you to iterate, since tg exposes that info.

Reply:
> https://core.telegram.org/bots/api#getuserprofilephotos
> If you're removing the offsets for the base function, might be worth adding a `GetProfilePhotosOffset(offset int, limit int)` which allows you to iterate, since tg exposes that info.

I'll check tomorrow morning

Reply:
please check newes commit